xrootd/xrdproto/protocol.go

```diff
@@ -44,6 +43,7 @@ type ServerErrorCode int32
 
 const (
 	InvalidRequestCode ServerErrorCode = 3006 // InvalidRequestCode indicates that request is invalid.
+	IOErrorCode        ServerErrorCode = 3007 // IOErrorCode indicates that IO error has occurred on the server side.
```

Comment:
`s/that IO error/that an IO error/`

also, it looks we are a bit inconsistent in how we name these error codes: either all of those should end in `...Code` or not.

looking at precedents in the Go stdlib (_e.g._ `net/http.StatusXYZ`), I guess we could drop the `Code` part...
(but this can be done in a later PR.)